babs/babs.py

```diff
@@ -3267,11 +3278,16 @@ def generate_job_submit_template(self, yaml_path, babs, system, test=False):
             eo_args = (
                 "-e "
                 + babs.analysis_path
-                + f"/logs/{job_name}.e%A "
+                + f"/logs/{job_name}.e%A_%a "
```

Comment:
Note: I personally like to have just one log file that has both the error and out together. That's a roadmap issue though